config/initializers/rollbar.rb

```diff
@@ -6,8 +6,8 @@
   config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
   config.environment = ENV['ROLLBAR_ENV']
 
-  # Here we'll disable in 'test':
-  config.enabled = false if Rails.env.test? || Rails.env.development?
+  # Here we'll disable in when not 'production':
```

Comment:
Hey @patybastos, thanks for the contribution!

One small nit: could this line say `Here we'll disable Rollbar when not in 'production':` instead?

Reply:
Hi @sman591 of course, I do it.